ngx-fudis/projects/ngx-fudis/src/lib/components/form/date/datepicker/datepicker.stories.ts

```diff
@@ -119,6 +99,7 @@ const ExampleTemplate: StoryFn = (args) => ({
 export const Datepicker = ExampleTemplate.bind({});
 Datepicker.args = {
   ...commonArgs,
+  message: 'This is custom string error message that is placed with content projection',
```

Comment:
I'm not sure why is this added here since it is only needed in the new DateFilter example. At least it shows unnecessarily in the controls: 

<img width="1175" height="262" alt="Image" src="https://github.com/user-attachments/assets/a0267da2-e21c-4bfd-b727-119e9a43194a" />